Packages/Primitives/TestKit/NFTAsset+PrimitivesTestKit.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+
+public extension NFTAsset {
```

Comment:
```suggestion
public extension NFTAsset {
```

avoid unique strings in mocks, just empty string is better